jaco_model/urdf/standalone_arm.xacro

```diff
@@ -17,7 +17,7 @@
 
   <xacro:include filename="$(find jaco_model)/urdf/jaco_arm.xacro" />
 
-<link name="jaco_api_origin">
+<link name="root">
```

Comment:
Why did you guys rename this from Kinova's version? It causes lots of errors because of a duplicate link?